src/oic/utils/sdb.py

```diff
@@ -1091,3 +1158,28 @@ def read(self, token):
             raise WrongTokenType("Not a grant token")
 
         return self._db[key]
+
+    def get_by_sub(self, sub):
+        return self._db.get_by_sub(sub)
+
+    def get_by_uid(self, uid):
+        return self._db.get_by_uid(uid)
+
+    def get_by_smid(self, smid):
+        return self._db.get_by_smid(smid)
+
+    def get_by_sub_and_client_id(self, sid, client_id):
+        return self._db.get_by_sub_and_x(sid, "client_id", client_id)
+
+    def get_by_sub_and_issuer(self, sid, issuer):
+        return self._db.get_by_sub_and_x(sid, "issuer", issuer)
+
+    def make_smid(self, sid):
+        """
+        Create a session management ID
+
+        :param sid:
+        :return: A session management ID
```

Comment:
This doesn't seem to be used (yet) is there a usage in the second part?

Reply:
Yes